.github/workflows/spring-boot-starter-camunda-sdk-ci.yml

```diff
@@ -98,6 +109,7 @@ jobs:
     outputs:
       flakyUnitTests: ${{ needs.unit-tests.outputs.flakyTests }}
     needs:
+      - build-zeebe
       - unit-tests
       - java-checks
```

Comment:
🔧 I might be wrong but I think this only ensures that all three are run before the `sdk-test-summary` runs. Since there is no dependency between those three, they will run in parallel. Thus the client is not installed into the local workflow run workspace before the other two jobs run.

I believe we must add such `needs` constraints to the other jobs that need the client to be built before they run. Plus, I'm wondering if the `checkout` action then also only needs to run once in the `build-zeebe` job. It's clearly been a too long since I've fumbled around with actions 😅